cves/kernel/CVE-2021-3490.yml

```
@@ -75,7 +81,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: ["https://bugzilla.suse.com/show_bug.cgi?id=1185641"]
```

Write just the bug id.
```suggestion
bugs: [1185641]
```